test/benchee/formatters/json_test.exs

```diff
@@ -1,144 +1,79 @@
 defmodule Benchee.Formatters.JSONTest do
   use ExUnit.Case
-  import ExUnit.CaptureIO
   doctest Benchee.Formatters.JSON
 
-  test ".output returns the suite again unchanged" do
```

Comment:
we seem to have lost this test?

Reply:
We no longer have `output/1` as a callback for formatters - just `format/2` and `write/2`, so I deleted the function and the associated test.

Reply:
ah right ase output is now assured in the formatter itself. Am/was tired and got ya, thanks!